scripts/deploy.sh

```diff
@@ -64,7 +69,7 @@ for git_ref in $(cat "${mydir}/../packages/hoprd/releases.json" | jq -r "to_entr
         version_maj_min=""
       fi
 
-      cluster_template_name="${cluster_name}-${package_version//./-}"
```

Comment:
This changes the semantic of cluster template names, meaning it will stay the same once it has been created once. Is that the intention?

Reply:
Yes, that was the intention, to align the instance-template name with the instance-group name